src/index.js

```diff
@@ -481,16 +488,44 @@ class EventsCoreAPI {
   }
 
   /**
-   * Authenticating events by verifying signature
+   * Authenticating events by verifying hmac signature
    *
    * @param {object} event JSON payload delivered to the registered webhook URL
    * @param {string} clientSecret Client secret can be retrieved from the Adobe I/O Console integration
-   * @param {string} signatureHeaderValue Value of x-adobe-signature header in each POST request to the registered webhook URL
+   * @param {string} deprecatedSignature Value of x-adobe-signature header in each POST request to the registered webhook URL
+   * @returns {boolean} If signature matches return true else return false
+   */
+  verifySignatureForEvent(event, clientSecret, deprecatedSignature) { 
+    if (clientSecret !== null && typeof (clientSecret) !== 'undefined') {
+      const hmacDigest = Base64.stringify(hmacSHA256(JSON.stringify(event), clientSecret))
+      return hmacDigest === deprecatedSignature
+    }
```

Comment:
What is returned if `clientSecret` is `null`/`undefined` ? Don't we need an `else` clause here, or better an early return?

Reply:
done